examples/bpk-component-segmented-control/examples.tsx

```diff
@@ -139,6 +135,31 @@ const ComplexSurfaceDefault = () => (
   />
 );
 
+const ComplexCanvasContrast = () => (
+  <BpkDarkExampleWrapper
+    padded
+    style={{ backgroundColor: privateSegmentedControlCanvasDefaultDay }}
```

Comment:
We want all the benefits of the Dark Wrapper but the Figma designs call for the examples to be on a lighter background